BZ2031163: change the storage location for Tang server keys (#40160)
openshift/openshift-docs: modules/nbde-backing-up-server-keys.adoc

[id="nbde-backing-up-server-keys_{context}"]
= Backing up keys for a Tang server

> The Tang server uses `/usr/libexec/tangd-keygen` to generate new keys and stores them in the `/var/db/tang` directory by default. Back up the contents of this directory to enable recovery in the event of the loss of the Tang server. The keys are sensitive and since they are able to perform the boot disk decryption of all hosts that have used them, the keys must be protected accordingly.

Reviewer: ```suggestion
The Tang server uses `/usr/libexec/tangd-keygen` to generate new keys and stores them in the `/var/db/tang` directory by default. To recover the Tang server in the event of a failure, back up this directory. The keys are sensitive and because they are able to perform the boot disk decryption of all hosts that have used them, the keys must be protected accordingly.
```
I know you are updating existing content. I am suggesting these changes.

Author: @mjpytlak Comment addressed. Please merge if it looks good. Thanks.
